### PR TITLE
Bugfix/1107/pattern/be

### PR DESCRIPTION
**Pull request by AlexanderBil**
## Description

When editing tapestries, added the ability to modify only one image in a form 

#### Screenshots

image
--------------
image

### Checklist
- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅All tests passed locally
- [x] ✨My changes working with up-to-date front-end and admin part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

**Review by maarekua — DISMISSED** [review]

File: modules/pattern/pattern.service.js [hunk 1]
```diff
@@ -137,14 +137,15 @@ class PatternsService {
...
     }
 
-    const uploadResult = await uploadService.uploadFile(image[0]);
-    const images = uploadResult.fileNames;
-    const constructorImg = await uploadSmallImage(image[1]);
-    pattern.constructorImg = constructorImg;
-
-    if (!images && constructorImg) {
-      return Pattern.findByIdAndUpdate(id, pattern).exec();
+    if (image[0].file) {
```
Comment (vasyaukr) [inline modules/pattern/pattern.service.js:140]: For my opinion it's bad naming... (image[1] && image[0].file) They are different type objects? Can u call them somehow?

File: modules/pattern/pattern.service.js [hunk 2]
```diff
@@ -137,14 +137,15 @@ class PatternsService {
...
+      const uploadResult = await uploadService.uploadFile(image[0]);
```
Comment (vasyaukr) [inline modules/pattern/pattern.service.js:141]: if we use uploadSmallImage, so then can we create another function for large image?

File: modules/pattern/pattern.service.js (hunk 1 above)
Comment (AlexanderBil) [reply modules/pattern/pattern.service.js]: changed to destructuring

File: modules/pattern/pattern.service.js (hunk 2 above)
Comment (AlexanderBil) [reply modules/pattern/pattern.service.js]: unfortunately using this function the page breaks

**Review by olga-gchak — APPROVED** [review]

**Review by AnastasiaKhukhra — APPROVED** [review]

**Pull request merged**